cmd/security-secrets-setup/edgex-stack-security.yml

```diff
@@ -0,0 +1,583 @@
+# /*******************************************************************************
+#  * Copyright 2019 Intel Corporation
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @author: Jim White, Dell
+#  * EdgeX Foundry, Edinburgh, version 1.0.1
```

Comment:
Per the discussion during core working group in the past, it is suggested to remove the version number from the header file. If a version number is needed, it is better to put into the file name.

Reply:
yes, this is just a testing example. will remove it